packages/ember-views/lib/mixins/action_support.js

```diff
@@ -131,8 +127,8 @@ export default Mixin.create({
     let action = this.actions && this.actions[actionName];
 
     if (action) {
-      let shouldBubble = action.apply(this, args) === true;
-      if (!shouldBubble) { return; }
+      let shouldBubble = action.apply(this, args);
+      if (shouldBubble === false) { return; }
```

Comment:
Before it was checking that the return value is exactly true, now it is checking that the return value is exactly false; if the function returns anything other than true or false, including returning nothing (undefined), the behavior is different